apps/src/context/climate-variable-provider.tsx

```diff
@@ -451,6 +456,39 @@ export const ClimateVariableProvider: React.FC<{
 		[dispatch]
 	);
 
+	const setForecastType = useCallback(
+		(forecastType: ForecastType) => {
+			dispatch(
+				updateClimateVariable({
```

Comment:
Question. If we need to add all dependencies, and functions we call also qualifies as a "*dependency*", shouldn't we also add `updateClimateVariable`?